sys-process/iotop/iotop-0.6.ebuild

```diff
@@ -33,3 +33,15 @@ PATCHES=(
 pkg_setup() {
 	linux-info_pkg_setup
 }
+
+pkg_postinst() {
+	cat << EOF | elog
+To enable the SWAPIN/IO columns Edit /etc/default/grub,
+adding "delayacct" as an option to the GRUB_CMDLINE_LINUX_DEFAULT
```

Comment:
grub is only one way to boot the Linux kernel. I think the first message should be "To enable the SWAPIN/IO columns, set the 'delayacct' kernel command-line parameter". Then you can go on with: "For example if using grub, set or append GRUB_CMDLINE_LINUX_DEFAULT with 'delayacct' in /etc/default/grub". Maybe a reference to the kernel's documentation on delay accounting can't hurt too: https://www.kernel.org/doc/html/latest/accounting/delay-accounting.html

Reply:
@Flowdalic Thanks for the review... I'm going update the PR with your suggestions.

Reply:
Done.
I think thjis should be ready.